package.json

```diff
@@ -63,6 +63,9 @@
     "zip": "tsx ./scripts/extension-zip.ts",
     "gen:vue-apis": "tsx ./scripts/vue-api-manifest.ts"
   },
+  "dependencies": {
+    "pathe": "^1.1.1"
```

Comment:
We should move the dep to the electron package.

Reply:
Yes, I just did lol